fem/bilininteg.hpp

```diff
@@ -63,6 +64,8 @@ class BilinearFormIntegrator : public NonlinearFormIntegrator
 
    virtual void AssemblePABoundaryFaces(const FiniteElementSpace &fes);
 
+   virtual void AssembleDiagonalNURBSPA(Vector &diag);
```

Comment:
Document that the diagonal is added to `diag` (copy comment from `AssembleDiagonalPA`).